### DISCUSSION TITLE
Looking for help...with our slogan.

### DISCUSSION THREAD

**Discussion by smolinari** [quasarframework/quasar#10895] [Ideas / Proposals]
Hey everyone,

We came up with a **slogan** for Quasar with the brand redesign and decided recently it **needs optimization/ rework**. 

The current slogan is:

> The open-source multi-platform framework based on Vuejs with an enterprise vocation

**We've come up with the new first part**

> The enterprise-grade cross-platform VueJS framework

**Now we're looking to the community for the next part. We are looking for one of two things. What quasar can do or who it is for,** 

For instance

> The enterprise-grade cross-platform VueJS framework for web professionals

or 

> The enterprise-grade cross-platform VueJS framework for modern front-end development

Write your own suggestions as answers below and up-vote a couple you also already like. I'll add the two above as entries too. 

Scott

**Comment by smolinari** [comment]
FIrst suggestion:

**The enterprise-grade cross-platform VueJS framework for web professionals**

Scott

**Reply by FrankySnow** [reply]
This slogan excludes non-professionals like me, who still want to use enterprise-grade software.

**Reply by IlCallo** [reply]
What in the slogan prevents you to use the framework for non-professional work 🤔

**Reply by smolinari** [reply]
@FrankySnow - Hopefully you have aspirations to do professional work? In other words, you can be a web professional and not actually do web work as a profession. 😁 That is what I am too. To be perfectly honest. 😁 It's hobby for me. 

Scott

**Reply by FrankySnow** [reply]
We must maybe define what a "web professional" is, because to me it is someone that does "web work as a profession".

All the web work I do is non-professional, so I do not consider myself as a web professional.

**Reply by Dirk-** [reply]
"for web professionals" sounds like a steep learning curve, which it doesn't require.

**Reply by smolinari** [reply]
@Dirk- Good point. It doesn't look like this slogan will be used anyway. 

Scott

**Comment by smolinari** [comment]
The second suggestion:

**The enterprise-grade cross-platform VueJS framework for modern front-end development**

Scott

**Comment by smolinari** [comment]
My suggestion: 😁 

**The enterprise-grade cross-platform VueJS framework for those who want to reach the stars**

Scott

**Reply by NorihisaItou** [reply]
>who want to reach the stars

[quasi-stellar object](https://en.wikipedia.org/wiki/Quasar)    ...  I'm trying to think of a cooler phrase!

**Reply by smolinari** [reply]
It is an [english idiom](https://idioms.thefreedictionary.com/Reach+for+the+stars!#:~:text=To%20set%20one's%20goals%20or,could%20do%20whatever%20you%20want.).

Scott

**Reply by NorihisaItou** [reply]
I am aware of that idiom and I think this is the best of the three you suggest.
However, I dared to try to make this better and tried to add some nuance of QSO...
Hmmm... but the sentence generated by my idea is going to be less than simple as a slogan...

Norihisa

**Comment by gruppler** [comment]
The enterprise-grade cross-platform VueJS framework for painless and efficient modern web development

**Reply by smolinari** [reply]
Not bad! 😃 

Scott

**Reply by gruppler** [reply]
Do one of those spinners to show different adjectives?
painless
efficient
enjoyable
hassle-free

**Reply by smolinari** [reply]
Oohh. I like that idea too. I wonder if a gif can be placed in our Social Media presences and it also work as an animated gif. 🤔 

Scott

**Reply by smolinari** [reply]
Something along these lines? https://codepen.io/smolinari/pen/eYRqNxx

Scott

**Reply by zanzara** [reply]
I would suggest **not** to use such adjectives, because they are relying in the eye of the beholder.
Particularly Q-beginners won't see this.... and this slogan is addressed to convince folks to start and use Q -- therefore they are beginners.
And if they might have a look first on the amount of issues, PRs and discussion points here... maybe they'd wonder about the slogan.

Btw.: Transferring Q1 to Q2 wasn't really 'hassle-free'  or 'painless'.

Don't get me wrong: I love Quasar. I use it and donate it.
But a slogan to get more new people convinced in using it, should be straight and short.

To me, it didn't need a slogan to switch to Quasar.  Examples and the awesome, perfect quasar.dev doc website had convinced me.

But I'll try to think about a slogan (I don't only want to ciritize it).

**Reply by zanzara** [reply]
> Something along these lines? https://codepen.io/smolinari/pen/eYRqNxx
> 
> Scott

This CSS is great for that purpose. Maybe even a tick faster ;-)

**Reply by smolinari** [reply]
@zanzara - It's faster now.

Scott

**Comment by ontwikkelfabriek** [comment]
@smolinari 

Not really on point but what about a comparison table between Quasar / Vuetify / other Vuejs frontend frameworks....

**Reply by smolinari** [reply]
@ontwikkelfabriek - We started one last year but quickly dropped it, as it was out of date shortly after we created it just for Quasar alone. So, we are avoiding the maintenance work of keeping such a comparison table up to date and we also want to avoid the crying of other maintainers, should we not depict their frameworks properly, because we forgot to "check" their work more often for changes/ improvements.

Scott

**Comment by ontwikkelfabriek** [comment]
@smolinari 

On point , the `one codebase to create all` argument won me over to try and stick to Quasar....

**Comment by kikan** [comment]
The enterprise-grade cross-platform VueJS framework which saves your time

**Comment by kikan** [comment]
The enterprise-grade cross-platform VueJS framework for hasslefree web dev

**Comment by pikil** [comment]
A single-codebase VueJS framework for web, mobile and desktop

**Reply by smolinari** [reply]
I forgot to mention, we are wanting to avoid anything that might infer "bloaty-ness". This seems to be often a misconception that because Quasar can do so much, the built apps are "bloated". Nothing can be further from the truth, but we still want to avoid that impression.

Scott

**Comment by Cafeinoman** [comment]
The enterprise-grade cross-platform VueJS framework, to Keep Internet Simple and Stable.

**Comment by matejmohar** [comment]
I would definitely try to not use complicated or less used words like "vocation" as some non-English speaking developers might not know the true meaning.  :)

**Comment by pikil** [comment]
A modern VueJS framework for personal and enterprise cross-platform projects

**Reply by smolinari** [reply]
Not bad. 👍 

Scott

**Comment by robfordww** [comment]
Quasar - It makes vue glow!

**Reply by smolinari** [reply]
That's a different take. 😄 

Scott

**Comment by Atom02** [comment]
Quasar - it's just works!

**Reply by smolinari** [reply]
Simple, but to the point. 

Everyone, don't forget to vote up others you like too!

Scott

**Comment by zmilan** [comment]
Enterprise-grade, Swiss knife, framework for cross-platform web development based on Vue.js

**Reply by IlCallo** [reply]
Unluckily, using `swiss knife` could lead to legal problems, so we're avoiding those particular words

**Reply by zmilan** [reply]
Oh, I didn't think about that... thanks for taking care about that

**Reply by zmilan** [reply]
What about: Enterprise-grade, battery included, framework for cross-platform web development based on Vue.js

**Comment by idc77** [comment]
"Quasar - Get things done"

Leave that enterprise out the door and especially stars and whatnot
I really hate to read overpromising marketing babble.
I'm no super-hero, and neither are you and your product won't make me one. (generally speaking)

If you have to add certain keywords like enterprise because of seo, add them in the description

**Reply by zmilan** [reply]
This "Get things done" is exactly why I enjoy using Quasar in the first place :)

**Reply by zmilan** [reply]
Quasar - Empowers everyone to get things done! Build on top of Vue.js.

**Reply by smolinari** [reply]
The "Get things done" phrase is more like a tagline than a slogan. Our tagline is currently "Beyond the Framework". The slogan can be a good bit longer. 😀 

Scott

**Reply by Dirk-** [reply]
> The "Get things done" phrase is more like a tagline than a slogan. Our tagline is currently "Beyond the Framework". The slogan can be a good bit longer. 😀

If you need another tagline in the future I would suggest "Consider It Done".

**Comment by jimoquinn** [comment]
Here is one:

*  Pierce the darkness - Quasar Framework

Couple more:


*  Illuminating web development - Quasar Framework
*  Light the path - Quasar Framework
*  Your ray of light - Quasar Framework

**Reply by smolinari** [reply]
@jiaxincui - Thanks, but see my comment about taglines and slogan length above. :)

Scott

**Comment by julie777** [comment]
This is tricky because quasar does two distinct things, and a few other on the side.
Easy multi-platform builds with very little effort.
Full set of quasar components base on Vuejs component, with styling support.

Quasar for VueJS - batteries included.

**Comment by etewiah** [comment]
Sorry, I can see problems with what I've come up with already but I'll just put it here in case it sparks some ideas:

Quasar, The enterprise-grade cross-platform VueJS framework with a constellation of ever expanding utilities.

Quasar has hit escape velocity.

I for one am really blown away by what quasar offers.  I am still learning but I hope I will soon be able to help promote it a lot more.

**Reply by smolinari** [reply]
Thanks for the input!

Scott

**Comment by suleimanas** [comment]
I think i'm late, but ..
With Quasar, it's hard to come up with a slogan.
so what if instead we list the main things that sold us Quasar at first then derive the slogan from that input. :)

**Reply by smolinari** [reply]
If not just to hear those things, please do? 😀 

Scott

**Reply by suleimanas** [reply]
As for me I do PWA and electron apps a lot, and i was searching for better way to smoothing the flow. the thing that hooked me was: 
1. How fast to create a new project
2. Professional PWA setup without wasting my time generating even the icons
3. It's Material design but the performance is much smoother than other frameworks, and OMG the table&select components (with Virtual scrolling) alone was PERFECT and incomparable to other framework
4. MORE AND MORE AND MORE
it's for sure unparalleled experience, Thank you guys

**Comment by customautosys** [comment]
"Cross platform apps as stunning as a Quasar"
